DigitalLearningSolutions.Web/Styles/shared/searchableElements/sort.scss

```diff
@@ -1,99 +1,81 @@
 @import "~nhsuk-frontend/packages/core/all";
 @import "common";
 
-.sort-dropdown {
-  border-radius: 4px;
+.sort-form {
+  width: 100%;
 }
 
-.sort-by-dropdown {
-  @extend .left-dropdown;
-}
-
-.sort-direction-dropdown {
-  @extend .input-with-no-js-submit-button;
+.sort-box-flex-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: flex-end;
+  width: 100%;
 
-  width: 63%;
+  .sort-by-dropdown {
+    @extend .left-dropdown;
+    display: flex;
 
-  @include mq($until: desktop) {
-    .js-enabled & {
-      width: 70%;
+    @include mq($until: tablet) {
+      width: 100%;
     }
-  }
-
-  @include mq($from: $xl-desktop) {
-    width: 74%;
-  }
-}
 
-.sort-direction-dropdown-container {
-  @include mq($from: desktop) {
-    width: calc(59% - 12px);
-    float: left;
+    @include mq($from: tablet) {
+      flex: 1;
+    }
   }
-}
 
-.sort-submit {
-  @extend .right-submit-button;
-}
+  .sort-order-button-flex-container {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
 
-.sort-row {
-  display: flex;
-  flex-direction: row;
+    .search-sort-flex-container & {
+      @include mq($from: tablet) {
+        width: 50%;
+      }
+    }

```

Comment:
What's this for?

Reply:
Differing layout of the sort dropdowns between the full width where they have the flex container, and 3/4 where they don't. Whether the styles are suitable I guess is a different matter.